core-feature-pack/common/src/main/resources/license/core-feature-pack-common-licenses.xml

```diff
@@ -230,6 +230,11 @@
           <url>http://www.apache.org/licenses/LICENSE-2.0</url>
           <distribution>repo</distribution>
         </license>
+        <license>
```

Comment:
@RanabirChakraborty , the same license needs to be added to wildfly-cli artifact in the same file:
https://github.com/wildfly/wildfly-core/blob/dd456d87dcbf03d6bbc3db05d48b1d7f38e92e0f/core-feature-pack/common/src/main/resources/license/core-feature-pack-common-licenses.xml#L582